outreach/hubs.md

```diff
@@ -0,0 +1,34 @@
+My name is [NAME], and I'm writing you on behalf of the RustBridge program. RustBridge is an inclusive and accessible event designed to teach newcomers to systems programming and programming in general about the programming language Rust. We, the [LOCAL ORGANIZING GROUP], are interested in reaching more learners, and would like to collaborate with you to host a free event for members of your institution and the general public.
+
+RustBridge follows BridgeFoundry format, which is used by RailsBridge and RubyBridge as well and has proven to be especially effective around learners who come from a diverse and non-traditional background. We have organized RustBridge events at universities and university-related events before, like the HackIllinois, University of Waterloo, the Canadian University Software Engineering Conference (CUSEC), and we think that it would be a great fit for [HUBNAME] as well.
+
+We are proposing would be an all-day event, meaning it would span about 6-8 hours.
+We have three goals for these kinds of events:
+
+1. to get learners up and running with a Rust development environment,
+2. to teach them the basics of the Rust language and ecosystem, and
+3. to provide them with their own personal and extendable project that they can work on after they finish.
+
+We make an extra effort to relate programming concepts in Rust to similar concepts in other languages, in order to make them easier to understand.
+
+Broadly, the event can be split into two parts: An interactive presentation about the core language concepts of Rust, using http://exercism.io as a platform. This way, learners can learn at their own pace using a well-known curriculum. The second part leads learners to build their own small website in Rust. This way, every learner can work at their own pace if they want, but still collaborate efficiently.
+
+### What We Offer (Incomplete)
+- Mentors/teachers for the workshop
+- Workshop materials
+- Food?
+- Spare laptops?
+- What's missing?
+
+### What We Require (Incomplete)
+- Venue (for how many learners?)
+- Main and standby power
+- Internet (minimum speed requirement?)
+- Spare laptops?
```

Comment:
not really, having a laptop is a hard requirement as a learner and shouldn't be offered from the venue